index.html

```diff
@@ -52,77 +65,108 @@ <h1 id="form-header">
           <label for="interests">Ich interessiere mich fuer ...*</label>
           <ul id="interests">
             <li>
-              <input type="checkbox" id="webdevopment" />
+              <input type="checkbox" id="webdevopment" name="webdevelopment" />
```

Comment:
The value="" attribute for the checkboxes in the interest section ist missing.

Reply:
It's okay to leave out values for checkboxes, if they have different names.